app/src/rightControlPanel/components/ControlPanel.jsx

```diff
@@ -125,39 +129,49 @@ class ControlPanel extends Component {
       </div >);
 
     const searchSubmenu = (
-      <SubMenu title="Vessels" icon={this.renderIcon('vessels')} extraHeader={searchHeader} onBack={this.onBack} >
+      <div>
+        <SubMenu
+          title="Vessels"
+          icon={this.renderIcon('vessels')}
+          extraHeader={searchHeader}
+          onBack={this.onBack}
+        >
+          {this.props.userPermissions !== null && this.props.userPermissions.indexOf('search') === -1 ?
+            <div >
+              <a
+                className="loginRequiredLink"
+                onClick={this.props.login}
+              >Only registered users can use the search feature. Click here to log in.</a >
+              <PinnedVesselList />
+            </div > :
+            <div >
+              <SearchPanel />
+              <PinnedVesselList />
+            </div >
+          }
+        </SubMenu >
         <VesselInfoPanel />
-        {this.props.userPermissions !== null && this.props.userPermissions.indexOf('search') === -1 ?
-          <div >
-            <a
-              className="loginRequiredLink"
-              onClick={this.props.login}
-            >Only registered users can use the search feature. Click here to log in.</a >
-            <PinnedVesselList />
-          </div > :
-          <div >
-            <SearchPanel />
-            <PinnedVesselList />
-          </div >
-        }
-      </SubMenu >
-    );
-
-    const basemapSubmenu = (
-      <SubMenu title="Basemap" icon={this.renderIcon('basemap')} onBack={this.onBack} >
-        <BasemapPanel />
-      </SubMenu >
+      </div>
     );
 
     const layerSubmenu = (
-      <SubMenu title="Layers" icon={this.renderIcon('layers')} onBack={this.onBack} >
+      <SubMenu
+        title="Layers"
+        icon={this.renderIcon('layers')}
+        onBack={this.onBack}
+        contentBefore={<BasemapPanel />}
```

Comment:
Its because of the styling. The margins in the Submenu content will make impossible to show the BasemapPanel as a block with 100% width of the submenu.